docs/source/data-proc-additional.ipynb

```diff
@@ -54,7 +54,68 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The _z_ axis is positive-down from the platform origin ${PCS}_o$, and all variables except for `vertical_offset` are defined with respect to ${PCS}_o$. All variables are in units meters.\n",
+        "\n",
+        "- `water_level`: Distance from the ${PCS}_o$ to the *nominal* (fixed) water level measured along the _z_-axis of the platform coordinate system (positive values are below the origin).\n",
+        "- `vertical_offset`: Time-varying distance from the *nominal* water level to the *actual* water level measured along the _z_-axis of the platform coordinate system (*positive values are when the actual water level is below the nominal water level*). For ships and similar platforms, this is *heave*, but the concept applies equally well to underwater vehicle depth.\n",
+        "- `MRU_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the motion reference unit (MRU) sensor origin.\n",
+        "- `position_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the latitude/longitude sensor origin.\n",
+        "- `transducer_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the sonar transducer.\n",
+        "\n",
+        "Using these variables, the depth of the transducer with respect to the actual water level can be calculated as:\n",
+        "\n",
+        "```\n",
+        "transducer_depth = transducer_offset_z - water_level - vertical_offset\n",
+        "```\n",
+        "\n",
+        "and the depth for a given echo sample is:\n",
+        "\n",
+        "```\n",
+        "sample_depth = transducer_depth + echo_range_z\n",
+        "```\n",
+        "\n",
+        "where `echo_range_z` is the _z_-axis component (positive down) of `echo_range`."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When the transducer points vertically (up or down), `echo_range_z = echo_range`. When the transducer is mounted with an angle to the vertical axis, the beam direction vector (defined by the variables `beam_direction_x`, `beam_direction_y`, `beam_direction_z` in the `Sonar/Beam_groupX`) should be used together with `echo_range` to calculate `echo_range_z`."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "For an echosounder in an underwater vehicle or moored on the seabed, a pressure sensor will provide the reference, time-varying depth. In this case:\n",
+        "- ${PCS}_o$ corresponds to the position of the pressure sensor\n",
+        "- `water_level` as defined by the convention is unnecessary and can be set to 0\n",
+        "- `vertical_offset = -pressure_sensor_depth`"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The sketch below shows the relationship between the above vertical coordinate (_z_-axis) variables.\n",
```

Comment:
How about moving up the figure and this sentence describing it to right after the sentence "where `echo_range_z` is the _z_-axis component (positive down) of `echo_range`."? That's where all the variables have been described conceptually. Everything below that sentence is a description of specific cases or additional calculations that may be needed.

Reply:
Sounds good. Please go ahead to make the change!

Reply:
Done in the latest commit.